superset/utils/core.py

```diff
@@ -1056,6 +1056,16 @@ def form_data_to_adhoc(form_data: Dict[str, Any], clause: str) -> AdhocFilterCla
     return result
 
 
+def append_filters_to_adhoc_filters(
```

Comment:
No place for reuse. We don't need to create functions.

Reply:
good point!